src/algo_tx.h

```diff
@@ -76,13 +76,13 @@ struct state_schema {
   uint64_t num_byteslice;
 };
 
-#define MAX_ACCT 2
+#define MAX_ACCT 4
 typedef uint8_t accounts_t[MAX_ACCT][32];
 
-#define MAX_ARG 2
+#define MAX_ARG 16
 #define MAX_ARGLEN 32
```

Comment:
Can this be increased?   32 bytes per argument is quite small.
NFDs for example allow users to set metadata into the state of a contract, and this means that the user won't be able to set any field longer than 32 characters with a ledger.  I'd like to see a bare-minimum of 128 bytes here as this corresponds at least to the limitation so current contract state keys.